mapbox-gl-native.pro

```diff
@@ -575,3 +574,11 @@ qtConfig(system-zlib) {
 } else {
     QT_PRIVATE += zlib-private
 }
+
+qtConfig(icu) {
```

Comment:
Remember to make sure we are linking with ICU when creating the plugin at `mapboxgl.pro`